src/main/java/io/cryostat/agent/ConfigModule.java

```diff
@@ -70,6 +75,7 @@ public abstract class ConfigModule {
             "cryostat.agent.webclient.connect.timeout-ms";
     public static final String CRYOSTAT_AGENT_WEBCLIENT_RESPONSE_TIMEOUT_MS =
             "cryostat.agent.webclient.response.timeout-ms";
+    public static final String CRYOSTAT_AGENT_TRUSTSTORES = "truststore.cert";
```

Comment:
Maybe this should be prefixed with `cryostat.agent` as well so that we minimize the risk that this clashes with the host application's configuration?

Reply:
Also, needs documentation in `README`